runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/process.json

```diff
@@ -320,6 +320,7 @@
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath": "Full path to file",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.WorksheetName": "Worksheet name",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.PersistChanges": "Persist changes",
+    "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath.Info": "It refers to the folder in the account specified on the bot",
```

Comment:
I would add an example what we expect from the user, because the input says "Full path to file". can be missleading and tbh I don't quite get what you mean by this message. Polish one doesn't help me 😄 
Imo all inputs should have a tooltip